components/button/test/button.visual-diff.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../../colors/colors.js';
```

Comment:
I don't think `d2l-colors` is needed here, or in any of the other button tests where you likely copied this from.

Reply:
@dlockhart are you guys good with me removing those references from the other buttons then, or do we use a "only alter code related to the U.S. which the PR is based on" kind of policy?

Reply:
I personally like the small, safe pull request strategy, but I'd be fine with cleaning up the other references here so as not to forget about it and since it's somewhat related.